src/aac/cli.py

```diff
@@ -42,20 +43,11 @@ def run_cli():
 
 def _validate_cmd(model_file: str):
     """The built-in validate command."""
+    with validation(parser.parse_file, model_file) as model:
+        if not model:
+            sys.exit(1)
```

Comment:
You're right that this pattern for error handling should be added to docs. So far, I added the documentation to [Plugins.md](https://github.com/jondavid-black/AaC/blob/bf05fc777376db8575c3b2e089b5be516390d8b1/docs/old/Plugins.md) since that specifically targets plugin developers. As a result, are you opposed to me adding it there, even if it doesn't stay there permanently?

I'm concerned that the number of people that actually read the README file will decrease proportionally to its length so I have been working on writing a story to split up the README such that users can go to different documentation that tells them specifically what they want to see. For example,

```markdown
## System Modelers

[... some brief description here ...]

For more information about modeling your system with the AaC language, see [here]().

## Plugin Developers

[... some brief description here ...]

For more information on developing plugins for the AaC tool, see [here]().
```

Reply:
I documented this in `Plugins.md` here: ae82e59. Let me know if you feel like it needs more visibility.